ui/recCardDisplay.ui

```diff
@@ -122,7 +122,6 @@
                 <property name="events">GDK_POINTER_MOTION_MASK | GDK_POINTER_MOTION_HINT_MASK | GDK_BUTTON_PRESS_MASK | GDK_BUTTON_RELEASE_MASK</property>
                 <property name="show_tabs">False</property>
                 <property name="show_border">False</property>
-                <property name="tab_hborder">1</property>
```

Comment:
In our collection of obsolete things: this `tab_hborder` property is no longer exists.